src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -677,7 +677,7 @@ private async Task<string> ComputePullRequestTitleAsync(InProgressPullRequest in
                 string baseTitle = $"[{targetBranch}] Update dependencies from";
                 StringBuilder titleBuilder = new StringBuilder(baseTitle);
                 bool prefixComma = false;
-                const int maxTitleLength = 80;
+                const int maxTitleLength = 150;
```

Comment:
Could you add a comment on why we picked 150?

Reply:
sure